mlflow/java/src/main/java/org/mlflow/sagemaker/ScoringServer.java

```diff
@@ -0,0 +1,270 @@
+package org.mlflow.sagemaker;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import org.mlflow.mleap.MLeapLoader;
+import org.mlflow.models.Model;
+import spark.Request;
+import spark.Response;
+import spark.Service;
+
+/** A RESTful webserver for {@link Predictor Predictors} that runs on the local host */
+public class ScoringServer {
+  public static final String RESPONSE_KEY_ERROR_MESSAGE = "Error";
+
+  public static final int HTTP_RESPONSE_CODE_SERVER_ERROR = 500;
+  public static final int HTTP_RESPONSE_CODE_SUCCESS = 200;
+  public static final int DEFAULT_PORT = 5001;
+
+  private enum RequestContentType {
+    Csv("text/csv"),
+    Json("application/json"),
+    Invalid("invalid");
```

Comment:
??

Reply:
In the `RequestContentType.fromValue` method, we assign this enum value to any string that does not match the `Csv` or `Json` values.